### ISSUE TITLE
Layered Navigation & Category Content Mode Enabled

### ISSUE THREAD

**Issue by mageho** [Smile-SA/elasticsuite#593]
### Preconditions

Webmin
Nginx
PHP-FPM

Magento Community Edition
Magento Version : 2.2.0

ElasticSuite Version : 2.4

Environment : Developper

Third party modules :
N/A

### Steps to reproduce

1. Catalog -> Inventory -> Categories 
2. Select any category in the category tree.
3. Enable Content Mode to reproduce this issue.
4. Go to this category in the frontend

### Expected result

1. Products List and Layered Navigation would be disabled and not displayed.

### Actual result

1. Products List is disabled but Layered Navigation is however displayed.

So far, we have hijacked this issue by one line in Smile_ElasticsuiteCatalog\Block\Navigation.php
Line 90
Function canShowBlock
```
if ($this->getLayer()->getCurrentCategory()->getDisplayMode() == \Magento\Catalog\Model\Category::DM_PAGE) {
        	return false;
 }
```

Could you take a look if you meet this problem too ?

Thank you,
Cheers,
Ilan

**Comment by mageho** [comment]
With this module, all catalog categories are in anchored mode by default. We can not customize layout categories by type anymore. Our template by default remove catalog.leftnav in catalog_category_view_type_default.xml but now, it is not used.

**Comment by afoucret** [comment]
@mageho : Anchored mode is disabled by design.

But you are right when sayinh there is bug occuring when the left col is displayed.
I think the proposed patch will break the filters display in search but it is a good start.

We will take a look at this one.

**Comment by afoucret** [comment]
PR #611 is merged on the branch 2.4.x.
Will be part of 2.4.0 release
